src/main/java/conversion/BuildVuServlet.java

```diff
@@ -139,13 +139,13 @@ protected void convert(Individual individual, Map<String, String[]> params,
     private void setErrorCode(final Individual individual, final int errorCode) {
         switch (errorCode) {
             case 1:
-                individual.setErrorCode(String.valueOf(1050)); // Libreoffice killed after 1 minute
+                individual.doError(1050); // Libreoffice killed after 1 minute
                 break;
             case 2:
-                individual.setErrorCode(String.valueOf(1070)); // Internal error
+                individual.doError(1050); // Internal error
```

Comment:
@EthanIDR - It seems you have used the same error code for each case. I have added the suggestion to correct this.

```suggestion
                individual.doError(1070); // Internal error
```